experiments/prediction/figure1.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+
+from RlGlue import RlGlue
+from utils.Collector import Collector
+from utils.policies import actionArrayToPolicy
+from utils.rl_glue import RlGlueCompatWrapper
+from utils.errors import buildRMSPBE
+
+from environments.RandomWalk import RandomWalk, Tabular, Dependent, Inverted
+from agents.TD import TD
+
+# --------------------------------
+# Set up parameters for experiment
+# --------------------------------
+
+RUNS = 10
+LEARNERS = [TD]
+
+PROBLEMS = [
```

Comment:
I guess this clears up the fact that random walk is an environment and tabular is the representation